src/components/MinimumDelegationAmountModal/MinimumDelegationAmountModal.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useCallback, useEffect } from 'react'
+import { ButtonType } from '@audius/stems'
+import BN from 'bn.js'
+
+import { useUpdateMinimumDelegationAmount } from 'store/actions/updateMinimumDelegationAmount'
+import Modal from 'components/Modal'
+import Button from 'components/Button'
+import TextField from 'components/TextField'
+import styles from './MinimumDelegationAmountModal.module.css'
+import { Status } from 'types'
+import ConfirmTransactionModal, {
+  StandaloneBox
+} from 'components/ConfirmTransactionModal'
+import { useModalControls } from 'utils/hooks'
+import DisplayAudio from 'components/DisplayAudio'
+import { TICKER } from 'utils/consts'
+import { checkWeiNumber, parseWeiNumber } from 'utils/numeric'
+
+const messages = {
+  title: 'Minimum Delegation Amount',
+  textLabel: 'MINIMUM DELEGATION AMOUNT',
+  currentLabel: 'Current: ',
+  btn: 'Change Minimum',
+  confirmChange: 'Confirm Change Minimum'
+}
+
+type OwnProps = {
+  minimumDelegationAmount: BN
+  isOpen: boolean
+  onClose: () => void
+}
+
+type MinimumDelegationAmountModalProps = OwnProps
+
+const MinimumDelegationAmountModal: React.FC<MinimumDelegationAmountModalProps> = ({
+  minimumDelegationAmount,
+  isOpen,
+  onClose
+}: MinimumDelegationAmountModalProps) => {
+  const [minDelegationBN, setMinDelegationBN] = useState(new BN('0'))
```

Comment:
shouldn't the default here be the network wide minimum?

Reply:
The user's minimum delegation amount is set to the on chain value while fetching users.
This amount here is the initial amount set in the modal shown while setting the new minimum. 
The text field is empty on modal open - the line below this one is the corresponding string value of the text input. 
We could set it to the minimum, but I don't think it matters too much. if anything we should show the minimum to the right of the input so users know with error checking. 

One thing to note is that there is no error checking on this input - lmk if we should add it w/ error text. but if the user tries to set the value to below the contract min value it will be set, but the logic when fetching the user's min always takes the greater value of network and user min.